src/main/java/org/myrobotlab/service/config/InMoov2Config.java

```diff
@@ -478,5 +478,27 @@
    */
   public double shutdownStartupSpeed = 50;
   
-  
+  // PEERS ????
```

Comment:
not boiler plate.. but ok.

Reply:
config related ... at some point we actually need to create a default InMoov config ... that works